Contour.m

```diff
@@ -0,0 +1,17 @@
+classdef Contour
+    % Represents a single tonal waveform
+    % Holds the raw frequency values as well as extrapolated variables
+
+    properties
+        frequencies % An array of raw frequency values
+        attributes  % A ContourAttributes instance
```

Comment:
Not sure if ARTwarp produces characteristics instances. 
Will we have to extract it ourselves? and if extracted, are they useful for the network? since ARTwarp didn't need these attributes